prov/efa/src/rxr/rxr_domain.c

```diff
@@ -328,6 +322,11 @@ int rxr_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 	rxr_domain->util_domain.mr_map.mode |=
 				OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_BASIC;
 
+	/*
+	 * Unset FI_MR_PROV_KEY bit, so that mr_map can use the requested key
```

Comment:
With this change, the comment in L#316 is stale (the reference to lower provider). Can you combing them and write a single comment explaining what line 322 and 328 are doing?